marketdata/stream/flow.go

```diff
@@ -175,7 +175,7 @@ func noSubscribeCallNecessary(trades, quotes, bars []string) bool {
 }
 
 func (c *client) writeSub(ctx context.Context) error {
-	msg, err := getSubChangeMessage(true, c.trades, c.quotes, c.bars)
+	msg, err := getSubChangeMessage(true, c.trades, c.quotes, c.bars, c.dailyBars)
```

Comment:
Please don't forget to update `noSubscribeCallNecessary` and `readSubResponse` in this file.

Reply:
Please also check `flow_test.go` whether there's something to update there (or add) or not.